tox.ini

```diff
@@ -40,12 +40,16 @@ commands =
       -W {posargs:--keep-going} \
       "{toxinidir}/docs/" "{toxinidir}/docs/html"
 
+    # Validate HTML
+    python validate-html.py "{toxinidir}/docs/html"
```

Comment:
I think it really should be in the docs building env, since that's the exact place where HTML files have been emitted and can be checked. I think I'll just remove the check. Feel free to implement this differently.

Reply:
FWIW the extension emitting these is one level deeper. That's why I wanted it there. Plus things like RTD invoke Sphinx directly meaning it'll run only if it's inside.